docs/rootfs-and-kernel-setup.md

```diff
@@ -116,7 +116,7 @@ Alpine Linux:
   
    # Then, copy the newly configured system to the rootfs image:
    for d in bin etc lib root sbin usr; do tar c "/$d" | tar x -C /my-rootfs; done
-   mkdir /my-rootfs/{dev,proc,run,sys,var}
```

Comment:
This bash syntax is valid. However, this is not valid sh syntax. Can you please clarify this in the patch description? Could you also please make the name of the patch more specific?